services/core/cards-consultation/src/main/java/org/opfab/cards/consultation/services/CardSubscription.java

```diff
@@ -204,13 +207,30 @@ private void registerUserListener(MessageListenerContainer mlc) {
      * <p>Constructs a non durable queue to exchange using queue name</p>
      * @return
      */
-    private Queue createQueue(String queueName, FanoutExchange exchange) {
+    private Queue createQueue(String queueName, FanoutExchange exchange, int retries, long retryInterval) {
         log.debug("CREATE queue for user {}", userLogin);
         Queue queue = QueueBuilder.nonDurable(queueName).build();
-        amqpAdmin.declareQueue(queue);
-
-        Binding binding = BindingBuilder.bind(queue).to(exchange);
-        amqpAdmin.declareBinding(binding);
+        boolean created = false;
+        int retry = 0;
+        while (!created) {
+            try {
+                amqpAdmin.declareQueue(queue);
+    
+                Binding binding = BindingBuilder.bind(queue).to(exchange);
+                amqpAdmin.declareBinding(binding);
+                created = true;
+            } catch (AmqpException ex) {
+                if (retry == retries) {
+                    throw ex;
+                }
+                try {
+                    Thread.sleep(retryInterval);
+                } catch (InterruptedException e) {
+                    Thread.currentThread().interrupt();
+                }
+                retry++;
```

Comment:
Add a log.info each time we do not succeed with the name of the queue and the retry number

Reply:
Done